app/javascript/vue.js

```diff
@@ -22,13 +26,17 @@ import SocialShareText from './components/social/SocialShareText'
 import Tooltip from './components/tooltip/Tooltip'
 import VTable from './components/table/VTable'
 
+// eventhub
+export const eventHub = new Vue()
```

Comment:
Do you prefer this to Vue.prototype.$eventHub = new Vue() (or however it's done in NatureMap)?

Reply:
Can we go with the former then - I find it nicer. Don't have to deal with imports.